db/schema.rb

```diff
@@ -332,8 +332,10 @@
     t.datetime "updated_at",                          :null => false
     t.text     "learner_url"
     t.boolean  "is_dirty",         :default => false
+    t.string   "key"
```

Comment:
Weird. I guess @dougmartin didn't include the changes to schema.rb when he did the interactive_run_states work.